src/honeycomb/opentelemetry/options.py

```diff
@@ -296,8 +297,14 @@ def __init__(
             _logger.warning(INVALID_EXPORTER_PROTOCOL_ERROR)
             self.metrics_exporter_protocol = exporter_protocol
 
-        # if http/protobuf protocol and using generic env or param
-        # append /v1/traces path
+        self.endpoint = os.environ.get(
+            HONEYCOMB_API_ENDPOINT,
+            None
+        )
+
+        if not self.endpoint:
+            self.endpoint = endpoint or DEFAULT_API_ENDPOINT
```

Comment:
Don't need the fallback from endpoint as you've already checked it's _falsey_ (eg None or empty) with the `if not`

```suggestion
            self.endpoint = DEFAULT_API_ENDPOINT
```

Reply:
I just moved it up into the first `self.endpoint` assignment :)